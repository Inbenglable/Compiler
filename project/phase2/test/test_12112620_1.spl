int foo(int i){
    if(i == 0){
        return 0;
    }
    else if(i == 1){
        return 1.1;
    }
    else{
        return '2';
    }
}