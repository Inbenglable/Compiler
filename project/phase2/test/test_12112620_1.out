Error type 8 at line 6: Return value type mismatches the declared type
Error type 8 at line 9: Return value type mismatches the declared type
